osu.Game.Rulesets.Osu/Difficulty/OsuPerformanceCalculator.cs

```diff
@@ -125,10 +123,8 @@ private double computeAimValue()
                 aimValue *= 1.0 + 0.04 * (12.0 - Attributes.ApproachRate);
             }
 
-            aimValue *= approachRateBonus;
-
-            // Scale the aim value with accuracy _slightly_.
-            aimValue *= 0.5 + accuracy / 2.0;
+            // Scale the aim value with accuracy
```

Comment:
Pointless comment. Please use comments to explain why, not to reword the operation on the next line. Might as well remove this if there's nothing to say